check/validation-chain-builders.js

```diff
@@ -1,6 +1,6 @@
-const check = require('./check');
+const checkModule = require('./check');
 
-const buildCheckFunction = locations => (fields, message) => check(fields, locations, message);
+const buildCheckFunction = locations => (fields, message, check = checkModule) => check(fields, locations, message);
```

Comment:
Did you mean to pass `check` to the outer function? `(locations, check = checkModule) => ...`?

Reply:
I'm not sure I understand the question. We need to use our custom version of check in the case that the user uses `createCheckAPI` so it is passed in as an argument. For the normal case, we just use the default argument checkModule for the normal non-custom check module.

If you look at `create-check-api.js` this is where we pass in our custom check function to `buildCheckFunction`.